src/pages/home/report/ReportActionCompose.js

```diff
@@ -496,7 +498,11 @@ class ReportActionCompose extends React.Component {
         const hasExceededMaxCommentLength = this.comment.length > CONST.MAX_COMMENT_LENGTH;
 
         return (
-            <View style={[shouldShowReportRecipientLocalTime && styles.chatItemComposeWithFirstRow, this.props.isComposerFullSize && styles.chatItemFullComposeRow]}>
+            <View style={[
+                shouldShowReportRecipientLocalTime && !this.props.network.isOffline && styles.chatItemComposeWithFirstRow,
+                this.props.isComposerFullSize && styles.chatItemFullComposeRow,
+            ]}
+            >
                 {shouldShowReportRecipientLocalTime
```

Comment:
Why this unproposed changes?

Reply:
I copied `ReportActionCompose.js` from the upstream and it has changed 
https://github.com/Expensify/App/blob/ba5a38f4a8f8eca2fdc63eb2b45aaccc85fe7997/src/pages/home/report/ReportActionCompose.js#L499-L503